app/src/main/java/com/futurice/freesound/feature/search/SearchActivity.java

```diff
@@ -160,4 +167,15 @@ private void addSearchFragment() {
                                    .commit();
     }
 
+    @Override
+    public void onDestroy() {
+        searchSnackbar.dismissSnackbar();
+        super.onDestroy();
+    }
+    public void showSnackbar(CharSequence charSequence){
```

Comment:
Choosing this one for both of you to see. Thanks for the feedback guys, I think I understood all the points and am going to rework them next chance I get!